src/community/backup-restore/core/src/main/java/org/geoserver/backuprestore/Backup.java

```diff
@@ -0,0 +1,481 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Copyright headers should only state 2016 Open Source Geospatial Foundation, no openplans mention, like in this file: https://github.com/geoserver/geoserver/blob/master/src/restconfig/src/test/java/org/geoserver/rest/resources/ResourceTest.java

Other (many? all?) files have the same issue, I'm not going to check and report each of them, basically the same advice applies to any new file (including spring context ones)